Userland/Libraries/LibGUI/TextEditor.h

```diff
@@ -252,6 +255,9 @@ class TextEditor
     virtual GUI::TextDocument& highlighter_did_request_document() final { return document(); }
     virtual GUI::TextPosition highlighter_did_request_cursor() const final { return m_cursor; }
 
+    // ^Clipboard::ClipboardClient
+    virtual void clipboard_content_did_change(String const& mime_type) override { m_paste_action->set_enabled(is_editable() && mime_type.starts_with("text/") && !Clipboard::the().data().is_empty()); }
```

Comment:
Please put this in the `.cpp` file instead. This is way too much random logic to have in a one-line override in the header :)

Reply:
Fixed, thank you! :^)